### PR TITLE
Task owner vs task requester doc

### PR DESCRIPTION
**Pull request by esthermmoturi**
@elvisdorkenoo and @dianabarsan Kindly let me know if the document improvement is understandable.I have also added a description of what guid means for those who are not familiar with the abbreviation.

### CODE REVIEW

File: content/en/contribute/docs/style-guide.md [hunk 1]
```diff
@@ -296,6 +296,11 @@ It is good practice for the image to link to the image file so that a larger ver
 |right-hand side|right hand side|
 |left-hand|left hand|
 
+### Where to place your images
+
+To ensure your images are loaded on the docsite in the right format,place them in a folder that is named similar to your .md file. For example in the scenario below, the image is linked in the style-guide.md file, thus it is placed in the style-guide folder.
```
Comment (dianabarsan) [inline content/en/contribute/docs/style-guide.md:301]:
```suggestion
To ensure your images are loaded on the docsite in the right format, place them in a folder that is named similar to your .md file. For example in the scenario below, the image is linked in the style-guide.md file, thus it is placed in the style-guide folder.
```

File: content/en/contribute/docs/style-guide.md [hunk 2]
```diff
@@ -296,6 +296,11 @@ It is good practice for the image to link to the image file so that a larger ver
...
+{{< figure src="where-to-place-images.png" link="where-to-place-images.png" >}}
```
Comment (dianabarsan) [inline content/en/contribute/docs/style-guide.md:302]: I wonder if we should be using proprietary image icons in this sample image. The `.md` files are associated with VS Code.

File: content/en/core/overview/db-schema.md [hunk 3]
```diff
@@ -257,12 +257,19 @@ Failed | Task was never terminated and the endDate has past
...
 user | The user settings id of the user who calculated and created the document. Used for controlling replication. (eg. `org.couchdb.user:agatha`)
-requester | The guid of the contact whose data brought about the creation of the document. Used for controlling cancellation.
+requester | The guid *(globally unique identifier that is an alphanumeric property identifying an object)* of the contact whose data brought about the creation of the document. Used for controlling cancellation. 
 owner | The guid of the contact whose profile this task will appear on in the contact's tab.
 forId | If completing a task's action opens a form. Completing the form creates a report. `forId` is the guid of the contact information that will be passed into the form. For most forms, the resulting report will be associated with this contact.
 emission | Minified task data emitted from the partner code.
 stateHistory | Each time the state attribute changes, the time of the change is recorded in the state history.
 
+
+To understand the difference between a task requester and a task owner, kindly see the example below:
```
Comment (dianabarsan) [inline content/en/core/overview/db-schema.md:267]: Thanks for adding this example :)

**Review by dianabarsan — APPROVED** [review]

**Review by elvisdorkenoo — APPROVED** [review]
LGTM 👍🏾

File: content/en/contribute/docs/style-guide.md (hunk 2 above)
Comment (esthermmoturi) [reply content/en/contribute/docs/style-guide.md:302]: I see what you mean, Ashley made this change. Is this okay?(https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#images)

Comment (dianabarsan) [reply content/en/contribute/docs/style-guide.md:302]: I'm not sure what I'm supposed to be looking at here.

Comment (dianabarsan) [reply content/en/contribute/docs/style-guide.md:302]: If it's about this image: https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/where-to-place-images.png

Then yes! No proprietary logos there!

**Pull request closed without merging**